hamocc/mo_param1_bgc.F90

```diff
@@ -484,7 +488,17 @@ subroutine init_indices()
       isssc13 = -1
       isssc14 = -1
     endif
-    nsedtra = i_sed_base + i_sed_cisonew
+
+    !NOTE: The age tracer currently always needs to be the last sedlay tracer
+    !      - otherwise issues in mo_sedshi.F90!
+    if (use_sediment_quality) then
+      i_sed_age   = 1
+      issso12_age = i_sed_base + i_sed_cisonew +1
+    else
+      i_sed_age   = 0
+      issso12_age = -1
+    endif
+    nsedtra = i_sed_base + i_sed_cisonew + i_sed_age
```

Comment:
Maybe define an additional counter, `nsedtra_noage = i_sed_base + i_sed_cisonew` for use in `mo_sedshi`?

Reply:
or define the age tracer as a separate array?

Reply:
I was thinking about adding an extra field, but it adds a lot of unnecessary code to introduce it. If it's ok, I consider Tomas idea - was also considering it before, but thought a local adjustment in sedshi is easier to follow (which is arguable).

Reply:
I'm fine with either solution, I don't think this is critical